src/hooks/useGetSourceParsedTokenAccounts.ts

```diff
@@ -640,18 +640,17 @@ const createNFTParsedTokenAccountFromCovalent = (
   covalent: CovalentData,
   nft_data: CovalentNFTData
 ): NFTParsedTokenAccount => {
+  const amount =
+    nft_data.token_balance == null
```

Comment:
I'd suggest going for `nft_data.token_balance ? formatUnits(nft_data.token_balance, covalent.contract_decimals) : "0"` to cover against any falsy value (e.g. `null`, `undefined`, `''`)